internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -145,6 +146,24 @@ func deployment(params manifestutils.Params) (*v1.Deployment, error) {
 							},
 							Resources:       manifestutils.Resources(tempo, manifestutils.QueryFrontendComponentName),
 							SecurityContext: manifestutils.TempoContainerSecurityContext(),
+							Env: []corev1.EnvVar{
+								{
+									Name:  "HTTP_CERT_FILE",
+									Value: " ",
```

Comment:
Does Tempo trim this whitespace in the env var and interpret it as a missing value?

Reply:
I had to put this whitespace because if not tempo will interpret the variable as not defined and set the default value, but this whitespace has the same effect as put nothing on the tls config, and disabled it.

May be what I can do is to get rid of default value and use env vars in all components, to make this thing uniform WDYT?

Reply:
> May be what I can do is to get rid of default value and use env vars in all components, to make this thing uniform WDYT?

Yeah, I think that would make it less hacky :)

I just remembered that if we want to have separate configs, we don't have to create two ConfigMaps, because a ConfigMap can have multiple keys. So alternatively we could just call `buildConfiguration(params)` [twice](https://github.com/grafana/tempo-operator/blob/2f1ffba9e35d1fe4f2b34381f8fc1886bfdd8600/internal/manifests/config/configmap.go#L22) with some new parameter, add `tempo-query-frontend.yaml` [here](https://github.com/grafana/tempo-operator/blob/2f1ffba9e35d1fe4f2b34381f8fc1886bfdd8600/internal/manifests/config/configmap.go#L40) and calculate the checksum over all files (tempo, tempo-query and tempo-query-frontend; not overrides.yaml, because tempo reloads that file automatically).

Reply:
Oh yes, you're right, may be is better to call buildConfiguration twice, for me sounds less hacky than the env vars solution. I'll modify this then.